.pre-commit-config.yaml

```diff
@@ -13,58 +13,58 @@ repos:
         types: [python]
         additional_dependencies:
           - "toml"
--   repo: https://github.com/pycqa/isort
+  - repo: https://github.com/pycqa/isort
     hooks:
       - id: isort
     rev: 5.10.1
-# -   repo: https://github.com/pycqa/pylint
-#     hooks:
-#         - id: pylint
-#           entry: pylint
-#           args: ['composer', 'examples', 'tests']
-#           language: python
-#           types: [python]
-#           require_serial: true
-#     rev: v2.12.2
--   repo: local
+  # -   repo: https://github.com/pycqa/pylint
```

Comment:
Curious about these formatting changes? Is this pre-commit running on itself? :rofl:

Reply:
Might be own VSCode yaml formatting